src/strategy/util.ts

```diff
@@ -1,5 +1,23 @@
 import * as murmurHash3 from 'murmurhash3js';
 
-export default function normalizedValue(id: string, groupId: string, normalizer = 100): number {
-  return (murmurHash3.x86.hash32(`${groupId}:${id}`) % normalizer) + 1;
+function normalizedValue(id: string,
+                         groupId: string,
+                         normalizer: number,
+                         seed = 0): number {
+  const hash = murmurHash3.x86.hash32(`${groupId}:${id}`, seed);
+  return (hash % normalizer) + 1;
+}
+
+const STRATEGY_SEED = 0;
+
+export function normalizedStrategyValue(id: string, groupId: string): number {
+  return normalizedValue(id, groupId, 100, STRATEGY_SEED);
```

Comment:
It looks like existing code always assumes 100 for strategies

Reply:
I believe we do, because the percentage slider for our gradualRollouts are 0-100 %